lib/beaker/host/unix/exec.rb

```diff
@@ -176,6 +176,8 @@ def ssh_service_restart
       exec(Beaker::Command.new("svcadm restart svc:/network/ssh:default"))
     when /(free|open)bsd/
       exec(Beaker::Command.new("sudo /etc/rc.d/sshd restart"))
+    else
+      raise ArgumentError.new("Unsupported Platform: '#{self['platform']}'")
```

Comment:
We don't have a style guide, so not a blocker, but this might be more idiomatic:

``` ruby
  raise ArgumentError, "Unsupported Platform: '#{self['platform']}'"
```